src/resources/app/components/Container/AnnouncementContainer.vue

```diff
@@ -21,7 +21,7 @@
 <script>
 import CreateAnnouncement from "../CreateAnnouncement";
 import AnnouncementCard from "../AnnouncementCard";
-import api from "../../../api/announcement/GetAnnouncements";
+import {getAnnouncementsAPI} from "../../../api/announcement";
```

Comment:
just my OCD,

```suggestion
import AnnouncementCard from "../AnnouncementCard";
import CreateAnnouncement from "../CreateAnnouncement";
import { getAnnouncementsAPI } from "../../../api/announcement";
```

I:
- reordered the import lexicographically
- added spaces between curly braces in the import statement